{
  "SchemaVersion": 2,
  "CreatedAt": "2025-09-08T14:22:24.642832307+02:00",
  "ArtifactName": "fairdatasystems/care:2025-09-08",
  "ArtifactType": "container_image",
  "Metadata": {
    "OS": {
      "Family": "alpine",
      "Name": "3.19.1"
    },
    "ImageID": "sha256:19de1bced3bde2f1f737ced57e1b96b7afa262b9a0afd19c55f677bee167de6d",
    "DiffIDs": [
      "sha256:d4fc045c9e3a848011de66f34b81f052d4f2c15a17bb196d637e526349601820",
      "sha256:4c9c2b9681abb404a680f956d25a7c0475872af4502ce47cdec2a2686f33d872",
      "sha256:80fef791f8cf38d8141a2963e4c83b8f5432c677137c657178d84c55f77e1476",
      "sha256:6c673d8c5e6c9efe162a8babbeaf3b6eac486ced1d36fcf33ae80388a6744c5c",
      "sha256:d46b5001af7f7573102bc997debe285c1d57558094abbd2d66b8399e1dea4a67",
      "sha256:7b0b5e1de3be9dfe8b9e1abfb50ea91838f9da16eb111a9ede1676cc8b856f7a",
      "sha256:90089e51e2e30c60e5183e41bc26a848c3ac0bcff306f9c25cdc7f26e4eb5a3c",
      "sha256:c69d247434760d91d4d502e8b709de64406a42a1402759a4d98a4486bad29c6d",
      "sha256:9f61c465811f24683fd257012cb50e8247502e86003da9bd7395ac246c3198e2"
    ],
    "RepoTags": [
      "fairdatasystems/care:2025-09-08"
    ],
    "RepoDigests": [
      "fairdatasystems/care@sha256:b17b4a15e08b253f261356726b1f445d686250d2c00001ce3b9fcba0f10786b6"
    ],
    "ImageConfig": {
      "architecture": "amd64",
      "container": "f10c9b6b8168e32d94b1dc494e378d2c00b773fa4f90caa767ca31a21000ffe0",
      "created": "2025-09-08T12:21:56.113049843Z",
      "docker_version": "26.1.3",
      "history": [
        {
          "created": "2024-01-27T00:30:48Z",
          "created_by": "/bin/sh -c #(nop) ADD file:37a76ec18f9887751cd8473744917d08b7431fc4085097bb6a09d81b41775473 in / "
        },
        {
          "created": "2024-01-27T00:30:48Z",
          "created_by": "/bin/sh -c #(nop)  CMD [\"/bin/sh\"]",
          "empty_layer": true
        },
        {
          "created": "2024-04-09T15:49:24Z",
          "created_by": "ENV PATH=/usr/local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
          "comment": "buildkit.dockerfile.v0",
          "empty_layer": true
        },
        {
          "created": "2024-04-09T15:49:24Z",
          "created_by": "ENV LANG=C.UTF-8",
          "comment": "buildkit.dockerfile.v0",
          "empty_layer": true
        },
        {
          "created": "2024-04-09T15:49:24Z",
          "created_by": "RUN /bin/sh -c set -eux; \tapk add --no-cache \t\tca-certificates \t\ttzdata \t; # buildkit",
          "comment": "buildkit.dockerfile.v0"
        },
        {
          "created": "2024-04-09T15:49:24Z",
          "created_by": "ENV GPG_KEY=7169605F62C751356D054A26A821E680E5FA6305",
          "comment": "buildkit.dockerfile.v0",
          "empty_layer": true
        },
        {
          "created": "2024-04-09T15:49:24Z",
          "created_by": "ENV PYTHON_VERSION=3.12.3",
          "comment": "buildkit.dockerfile.v0",
          "empty_layer": true
        },
        {
          "created": "2024-04-09T15:49:24Z",
          "created_by": "RUN /bin/sh -c set -eux; \t\tapk add --no-cache --virtual .build-deps \t\tgnupg \t\ttar \t\txz \t\t\t\tbluez-dev \t\tbzip2-dev \t\tdpkg-dev dpkg \t\texpat-dev \t\tfindutils \t\tgcc \t\tgdbm-dev \t\tlibc-dev \t\tlibffi-dev \t\tlibnsl-dev \t\tlibtirpc-dev \t\tlinux-headers \t\tmake \t\tncurses-dev \t\topenssl-dev \t\tpax-utils \t\treadline-dev \t\tsqlite-dev \t\ttcl-dev \t\ttk \t\ttk-dev \t\tutil-linux-dev \t\txz-dev \t\tzlib-dev \t; \t\twget -O python.tar.xz \"https://www.python.org/ftp/python/${PYTHON_VERSION%%[a-z]*}/Python-$PYTHON_VERSION.tar.xz\"; \twget -O python.tar.xz.asc \"https://www.python.org/ftp/python/${PYTHON_VERSION%%[a-z]*}/Python-$PYTHON_VERSION.tar.xz.asc\"; \tGNUPGHOME=\"$(mktemp -d)\"; export GNUPGHOME; \tgpg --batch --keyserver hkps://keys.openpgp.org --recv-keys \"$GPG_KEY\"; \tgpg --batch --verify python.tar.xz.asc python.tar.xz; \tgpgconf --kill all; \trm -rf \"$GNUPGHOME\" python.tar.xz.asc; \tmkdir -p /usr/src/python; \ttar --extract --directory /usr/src/python --strip-components=1 --file python.tar.xz; \trm python.tar.xz; \t\tcd /usr/src/python; \tgnuArch=\"$(dpkg-architecture --query DEB_BUILD_GNU_TYPE)\"; \t./configure \t\t--build=\"$gnuArch\" \t\t--enable-loadable-sqlite-extensions \t\t--enable-optimizations \t\t--enable-option-checking=fatal \t\t--enable-shared \t\t--with-lto \t\t--with-system-expat \t\t--without-ensurepip \t; \tnproc=\"$(nproc)\"; \tEXTRA_CFLAGS=\"-DTHREAD_STACK_SIZE=0x100000\"; \tLDFLAGS=\"${LDFLAGS:--Wl},--strip-all\"; \tmake -j \"$nproc\" \t\t\"EXTRA_CFLAGS=${EXTRA_CFLAGS:-}\" \t\t\"LDFLAGS=${LDFLAGS:-}\" \t\t\"PROFILE_TASK=${PROFILE_TASK:-}\" \t; \trm python; \tmake -j \"$nproc\" \t\t\"EXTRA_CFLAGS=${EXTRA_CFLAGS:-}\" \t\t\"LDFLAGS=${LDFLAGS:--Wl},-rpath='\\$\\$ORIGIN/../lib'\" \t\t\"PROFILE_TASK=${PROFILE_TASK:-}\" \t\tpython \t; \tmake install; \t\tcd /; \trm -rf /usr/src/python; \t\tfind /usr/local -depth \t\t\\( \t\t\t\\( -type d -a \\( -name test -o -name tests -o -name idle_test \\) \\) \t\t\t-o \\( -type f -a \\( -name '*.pyc' -o -name '*.pyo' -o -name 'libpython*.a' \\) \\) \t\t\\) -exec rm -rf '{}' + \t; \t\tfind /usr/local -type f -executable -not \\( -name '*tkinter*' \\) -exec scanelf --needed --nobanner --format '%n#p' '{}' ';' \t\t| tr ',' '\\n' \t\t| sort -u \t\t| awk 'system(\"[ -e /usr/local/lib/\" $1 \" ]\") == 0 { next } { print \"so:\" $1 }' \t\t| xargs -rt apk add --no-network --virtual .python-rundeps \t; \tapk del --no-network .build-deps; \t\tpython3 --version # buildkit",
          "comment": "buildkit.dockerfile.v0"
        },
        {
          "created": "2024-04-09T15:49:24Z",
          "created_by": "RUN /bin/sh -c set -eux; \tfor src in idle3 pydoc3 python3 python3-config; do \t\tdst=\"$(echo \"$src\" | tr -d 3)\"; \t\t[ -s \"/usr/local/bin/$src\" ]; \t\t[ ! -e \"/usr/local/bin/$dst\" ]; \t\tln -svT \"$src\" \"/usr/local/bin/$dst\"; \tdone # buildkit",
          "comment": "buildkit.dockerfile.v0"
        },
        {
          "created": "2024-04-09T15:49:24Z",
          "created_by": "ENV PYTHON_PIP_VERSION=24.0",
          "comment": "buildkit.dockerfile.v0",
          "empty_layer": true
        },
        {
          "created": "2024-04-09T15:49:24Z",
          "created_by": "ENV PYTHON_GET_PIP_URL=https://github.com/pypa/get-pip/raw/dbf0c85f76fb6e1ab42aa672ffca6f0a675d9ee4/public/get-pip.py",
          "comment": "buildkit.dockerfile.v0",
          "empty_layer": true
        },
        {
          "created": "2024-04-09T15:49:24Z",
          "created_by": "ENV PYTHON_GET_PIP_SHA256=dfe9fd5c28dc98b5ac17979a953ea550cec37ae1b47a5116007395bfacff2ab9",
          "comment": "buildkit.dockerfile.v0",
          "empty_layer": true
        },
        {
          "created": "2024-04-09T15:49:24Z",
          "created_by": "RUN /bin/sh -c set -eux; \t\twget -O get-pip.py \"$PYTHON_GET_PIP_URL\"; \techo \"$PYTHON_GET_PIP_SHA256 *get-pip.py\" | sha256sum -c -; \t\texport PYTHONDONTWRITEBYTECODE=1; \t\tpython get-pip.py \t\t--disable-pip-version-check \t\t--no-cache-dir \t\t--no-compile \t\t\"pip==$PYTHON_PIP_VERSION\" \t; \trm -f get-pip.py; \t\tpip --version # buildkit",
          "comment": "buildkit.dockerfile.v0"
        },
        {
          "created": "2024-04-09T15:49:24Z",
          "created_by": "CMD [\"python3\"]",
          "comment": "buildkit.dockerfile.v0",
          "empty_layer": true
        },
        {
          "created": "2025-07-04T16:14:19Z",
          "created_by": "WORKDIR /code",
          "comment": "buildkit.dockerfile.v0"
        },
        {
          "created": "2025-07-31T16:10:55Z",
          "created_by": "COPY . /code # buildkit",
          "comment": "buildkit.dockerfile.v0"
        },
        {
          "created": "2025-07-31T16:11:10Z",
          "created_by": "RUN /bin/sh -c pip install --no-cache-dir --upgrade -r /code/requirements.txt # buildkit",
          "comment": "buildkit.dockerfile.v0"
        },
        {
          "created": "2025-07-31T16:11:10Z",
          "created_by": "CMD [\"uvicorn\" \"--host\" \"0.0.0.0\" \"main:app\"]",
          "comment": "buildkit.dockerfile.v0",
          "empty_layer": true
        },
        {
          "created": "2025-09-08T12:21:56Z",
          "created_by": "uvicorn --host 0.0.0.0 main:app"
        }
      ],
      "os": "linux",
      "rootfs": {
        "type": "layers",
        "diff_ids": [
          "sha256:d4fc045c9e3a848011de66f34b81f052d4f2c15a17bb196d637e526349601820",
          "sha256:4c9c2b9681abb404a680f956d25a7c0475872af4502ce47cdec2a2686f33d872",
          "sha256:80fef791f8cf38d8141a2963e4c83b8f5432c677137c657178d84c55f77e1476",
          "sha256:6c673d8c5e6c9efe162a8babbeaf3b6eac486ced1d36fcf33ae80388a6744c5c",
          "sha256:d46b5001af7f7573102bc997debe285c1d57558094abbd2d66b8399e1dea4a67",
          "sha256:7b0b5e1de3be9dfe8b9e1abfb50ea91838f9da16eb111a9ede1676cc8b856f7a",
          "sha256:90089e51e2e30c60e5183e41bc26a848c3ac0bcff306f9c25cdc7f26e4eb5a3c",
          "sha256:c69d247434760d91d4d502e8b709de64406a42a1402759a4d98a4486bad29c6d",
          "sha256:9f61c465811f24683fd257012cb50e8247502e86003da9bd7395ac246c3198e2"
        ]
      },
      "config": {
        "Cmd": [
          "uvicorn",
          "--host",
          "0.0.0.0",
          "main:app"
        ],
        "Env": [
          "PATH=/usr/local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
          "LANG=C.UTF-8",
          "GPG_KEY=7169605F62C751356D054A26A821E680E5FA6305",
          "PYTHON_VERSION=3.12.3",
          "PYTHON_PIP_VERSION=24.0",
          "PYTHON_GET_PIP_URL=https://github.com/pypa/get-pip/raw/dbf0c85f76fb6e1ab42aa672ffca6f0a675d9ee4/public/get-pip.py",
          "PYTHON_GET_PIP_SHA256=dfe9fd5c28dc98b5ac17979a953ea550cec37ae1b47a5116007395bfacff2ab9"
        ],
        "Hostname": "f10c9b6b8168",
        "Image": "pabloalarconm/care-sm-toolkit:0.3.0",
        "WorkingDir": "/code"
      }
    }
  },
  "Results": [
    {
      "Target": "fairdatasystems/care:2025-09-08 (alpine 3.19.1)",
      "Class": "os-pkgs",
      "Type": "alpine",
      "Vulnerabilities": [
        {
          "VulnerabilityID": "CVE-2024-6119",
          "PkgID": "libcrypto3@3.1.4-r6",
          "PkgName": "libcrypto3",
          "PkgIdentifier": {
            "PURL": "pkg:apk/alpine/libcrypto3@3.1.4-r6?arch=x86_64\u0026distro=3.19.1"
          },
          "InstalledVersion": "3.1.4-r6",
          "FixedVersion": "3.1.7-r0",
          "Status": "fixed",
          "Layer": {
            "DiffID": "sha256:80fef791f8cf38d8141a2963e4c83b8f5432c677137c657178d84c55f77e1476"
          },
          "SeveritySource": "nvd",
          "PrimaryURL": "https://avd.aquasec.com/nvd/cve-2024-6119",
          "DataSource": {
            "ID": "alpine",
            "Name": "Alpine Secdb",
            "URL": "https://secdb.alpinelinux.org/"
          },
          "Title": "openssl: Possible denial of service in X.509 name checks",
          "Description": "Issue summary: Applications performing certificate name checks (e.g., TLS\nclients checking server certificates) may attempt to read an invalid memory\naddress resulting in abnormal termination of the application process.\n\nImpact summary: Abnormal termination of an application can a cause a denial of\nservice.\n\nApplications performing certificate name checks (e.g., TLS clients checking\nserver certificates) may attempt to read an invalid memory address when\ncomparing the expected name with an `otherName` subject alternative name of an\nX.509 certificate. This may result in an exception that terminates the\napplication program.\n\nNote that basic certificate chain validation (signatures, dates, ...) is not\naffected, the denial of service can occur only when the application also\nspecifies an expected DNS name, Email address or IP address.\n\nTLS servers rarely solicit client certificates, and even when they do, they\ngenerally don't perform a name check against a reference identifier (expected\nidentity), but rather extract the presented identity after checking the\ncertificate chain.  So TLS servers are generally not affected and the severity\nof the issue is Moderate.\n\nThe FIPS modules in 3.3, 3.2, 3.1 and 3.0 are not affected by this issue.",
          "Severity": "HIGH",
          "CweIDs": [
            "CWE-843"
          ],
          "VendorSeverity": {
            "alma": 2,
            "amazon": 2,
            "azure": 3,
            "cbl-mariner": 3,
            "nvd": 3,
            "oracle-oval": 2,
            "photon": 3,
            "redhat": 2,
            "rocky": 2,
            "ubuntu": 2
          },
          "CVSS": {
            "nvd": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
              "V3Score": 7.5
            },
            "redhat": {
              "V3Vector": "CVSS:3.1/AV:N/AC:H/PR:N/UI:N/S:U/C:N/I:N/A:H",
              "V3Score": 5.9
            }
          },
          "References": [
            "http://www.openwall.com/lists/oss-security/2024/09/03/4",
            "https://access.redhat.com/errata/RHSA-2024:8935",
            "https://access.redhat.com/security/cve/CVE-2024-6119",
            "https://bugzilla.redhat.com/2306158",
            "https://bugzilla.redhat.com/show_bug.cgi?id=2306158",
            "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2024-6119",
            "https://errata.almalinux.org/9/ALSA-2024-8935.html",
            "https://errata.rockylinux.org/RLSA-2024:6783",
            "https://github.com/openssl/openssl/commit/05f360d9e849a1b277db628f1f13083a7f8dd04f",
            "https://github.com/openssl/openssl/commit/06d1dc3fa96a2ba5a3e22735a033012aadc9f0d6",
            "https://github.com/openssl/openssl/commit/621f3729831b05ee828a3203eddb621d014ff2b2",
            "https://github.com/openssl/openssl/commit/7dfcee2cd2a63b2c64b9b4b0850be64cb695b0a0",
            "https://github.com/openssl/openssl/security/advisories/GHSA-5qrj-vq78-58fj",
            "https://linux.oracle.com/cve/CVE-2024-6119.html",
            "https://linux.oracle.com/errata/ELSA-2024-8935.html",
            "https://lists.freebsd.org/archives/freebsd-security/2024-September/000303.html",
            "https://nvd.nist.gov/vuln/detail/CVE-2024-6119",
            "https://openssl-library.org/news/secadv/20240903.txt",
            "https://security.netapp.com/advisory/ntap-20240912-0001/",
            "https://ubuntu.com/security/notices/USN-6986-1",
            "https://www.cve.org/CVERecord?id=CVE-2024-6119"
          ],
          "PublishedDate": "2024-09-03T16:15:07.177Z",
          "LastModifiedDate": "2025-06-03T10:51:54.117Z"
        },
        {
          "VulnerabilityID": "CVE-2024-45491",
          "PkgID": "libexpat@2.6.2-r0",
          "PkgName": "libexpat",
          "PkgIdentifier": {
            "PURL": "pkg:apk/alpine/libexpat@2.6.2-r0?arch=x86_64\u0026distro=3.19.1"
          },
          "InstalledVersion": "2.6.2-r0",
          "FixedVersion": "2.6.3-r0",
          "Status": "fixed",
          "Layer": {
            "DiffID": "sha256:80fef791f8cf38d8141a2963e4c83b8f5432c677137c657178d84c55f77e1476"
          },
          "SeveritySource": "nvd",
          "PrimaryURL": "https://avd.aquasec.com/nvd/cve-2024-45491",
          "DataSource": {
            "ID": "alpine",
            "Name": "Alpine Secdb",
            "URL": "https://secdb.alpinelinux.org/"
          },
          "Title": "libexpat: Integer Overflow or Wraparound",
          "Description": "An issue was discovered in libexpat before 2.6.3. dtdCopy in xmlparse.c can have an integer overflow for nDefaultAtts on 32-bit platforms (where UINT_MAX equals SIZE_MAX).",
          "Severity": "CRITICAL",
          "CweIDs": [
            "CWE-190"
          ],
          "VendorSeverity": {
            "alma": 2,
            "azure": 4,
            "cbl-mariner": 4,
            "nvd": 4,
            "oracle-oval": 2,
            "photon": 4,
            "redhat": 2,
            "rocky": 2,
            "ubuntu": 2
          },
          "CVSS": {
            "nvd": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H",
              "V3Score": 9.8
            },
            "redhat": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
              "V3Score": 7.5
            }
          },
          "References": [
            "https://access.redhat.com/errata/RHSA-2024:6754",
            "https://access.redhat.com/security/cve/CVE-2024-45491",
            "https://bugzilla.redhat.com/2308615",
            "https://bugzilla.redhat.com/2308616",
            "https://bugzilla.redhat.com/2308617",
            "https://bugzilla.redhat.com/show_bug.cgi?id=2308615",
            "https://bugzilla.redhat.com/show_bug.cgi?id=2308616",
            "https://bugzilla.redhat.com/show_bug.cgi?id=2308617",
            "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2024-45490",
            "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2024-45491",
            "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2024-45492",
            "https://errata.almalinux.org/9/ALSA-2024-6754.html",
            "https://errata.rockylinux.org/RLSA-2024:6754",
            "https://github.com/libexpat/libexpat/blob/R_2_6_3/expat/Changes",
            "https://github.com/libexpat/libexpat/issues/888",
            "https://github.com/libexpat/libexpat/pull/891",
            "https://linux.oracle.com/cve/CVE-2024-45491.html",
            "https://linux.oracle.com/errata/ELSA-2024-8859.html",
            "https://nvd.nist.gov/vuln/detail/CVE-2024-45491",
            "https://security.netapp.com/advisory/ntap-20241018-0003/",
            "https://ubuntu.com/security/notices/USN-7000-1",
            "https://ubuntu.com/security/notices/USN-7000-2",
            "https://ubuntu.com/security/notices/USN-7001-1",
            "https://ubuntu.com/security/notices/USN-7001-2",
            "https://www.cve.org/CVERecord?id=CVE-2024-45491"
          ],
          "PublishedDate": "2024-08-30T03:15:03.85Z",
          "LastModifiedDate": "2024-11-21T09:37:51.08Z"
        },
        {
          "VulnerabilityID": "CVE-2024-45492",
          "PkgID": "libexpat@2.6.2-r0",
          "PkgName": "libexpat",
          "PkgIdentifier": {
            "PURL": "pkg:apk/alpine/libexpat@2.6.2-r0?arch=x86_64\u0026distro=3.19.1"
          },
          "InstalledVersion": "2.6.2-r0",
          "FixedVersion": "2.6.3-r0",
          "Status": "fixed",
          "Layer": {
            "DiffID": "sha256:80fef791f8cf38d8141a2963e4c83b8f5432c677137c657178d84c55f77e1476"
          },
          "SeveritySource": "nvd",
          "PrimaryURL": "https://avd.aquasec.com/nvd/cve-2024-45492",
          "DataSource": {
            "ID": "alpine",
            "Name": "Alpine Secdb",
            "URL": "https://secdb.alpinelinux.org/"
          },
          "Title": "libexpat: integer overflow",
          "Description": "An issue was discovered in libexpat before 2.6.3. nextScaffoldPart in xmlparse.c can have an integer overflow for m_groupSize on 32-bit platforms (where UINT_MAX equals SIZE_MAX).",
          "Severity": "CRITICAL",
          "CweIDs": [
            "CWE-190"
          ],
          "VendorSeverity": {
            "alma": 2,
            "azure": 4,
            "cbl-mariner": 4,
            "nvd": 4,
            "oracle-oval": 2,
            "photon": 4,
            "redhat": 2,
            "rocky": 2,
            "ubuntu": 2
          },
          "CVSS": {
            "nvd": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H",
              "V3Score": 9.8
            },
            "redhat": {
              "V3Vector": "CVSS:3.1/AV:L/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
              "V3Score": 6.2
            }
          },
          "References": [
            "https://access.redhat.com/errata/RHSA-2024:6754",
            "https://access.redhat.com/security/cve/CVE-2024-45492",
            "https://bugzilla.redhat.com/2308615",
            "https://bugzilla.redhat.com/2308616",
            "https://bugzilla.redhat.com/2308617",
            "https://bugzilla.redhat.com/show_bug.cgi?id=2308615",
            "https://bugzilla.redhat.com/show_bug.cgi?id=2308616",
            "https://bugzilla.redhat.com/show_bug.cgi?id=2308617",
            "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2024-45490",
            "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2024-45491",
            "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2024-45492",
            "https://errata.almalinux.org/9/ALSA-2024-6754.html",
            "https://errata.rockylinux.org/RLSA-2024:6754",
            "https://github.com/libexpat/libexpat/blob/R_2_6_3/expat/Changes",
            "https://github.com/libexpat/libexpat/issues/889",
            "https://github.com/libexpat/libexpat/pull/892",
            "https://linux.oracle.com/cve/CVE-2024-45492.html",
            "https://linux.oracle.com/errata/ELSA-2024-6989.html",
            "https://nvd.nist.gov/vuln/detail/CVE-2024-45492",
            "https://security.netapp.com/advisory/ntap-20241018-0005/",
            "https://ubuntu.com/security/notices/USN-7000-1",
            "https://ubuntu.com/security/notices/USN-7000-2",
            "https://www.cve.org/CVERecord?id=CVE-2024-45492"
          ],
          "PublishedDate": "2024-08-30T03:15:03.93Z",
          "LastModifiedDate": "2024-11-21T09:37:51.307Z"
        },
        {
          "VulnerabilityID": "CVE-2024-45490",
          "PkgID": "libexpat@2.6.2-r0",
          "PkgName": "libexpat",
          "PkgIdentifier": {
            "PURL": "pkg:apk/alpine/libexpat@2.6.2-r0?arch=x86_64\u0026distro=3.19.1"
          },
          "InstalledVersion": "2.6.2-r0",
          "FixedVersion": "2.6.3-r0",
          "Status": "fixed",
          "Layer": {
            "DiffID": "sha256:80fef791f8cf38d8141a2963e4c83b8f5432c677137c657178d84c55f77e1476"
          },
          "SeveritySource": "nvd",
          "PrimaryURL": "https://avd.aquasec.com/nvd/cve-2024-45490",
          "DataSource": {
            "ID": "alpine",
            "Name": "Alpine Secdb",
            "URL": "https://secdb.alpinelinux.org/"
          },
          "Title": "libexpat: Negative Length Parsing Vulnerability in libexpat",
          "Description": "An issue was discovered in libexpat before 2.6.3. xmlparse.c does not reject a negative length for XML_ParseBuffer.",
          "Severity": "HIGH",
          "CweIDs": [
            "CWE-611",
            "CWE-190"
          ],
          "VendorSeverity": {
            "alma": 2,
            "amazon": 3,
            "azure": 3,
            "cbl-mariner": 3,
            "nvd": 3,
            "oracle-oval": 2,
            "photon": 3,
            "redhat": 2,
            "rocky": 2,
            "ubuntu": 2
          },
          "CVSS": {
            "nvd": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
              "V3Score": 7.5
            },
            "redhat": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
              "V3Score": 7.5
            }
          },
          "References": [
            "https://access.redhat.com/errata/RHSA-2024:6754",
            "https://access.redhat.com/security/cve/CVE-2024-45490",
            "https://bugzilla.redhat.com/2308615",
            "https://bugzilla.redhat.com/2308616",
            "https://bugzilla.redhat.com/2308617",
            "https://bugzilla.redhat.com/show_bug.cgi?id=2308615",
            "https://bugzilla.redhat.com/show_bug.cgi?id=2308616",
            "https://bugzilla.redhat.com/show_bug.cgi?id=2308617",
            "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2024-45490",
            "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2024-45491",
            "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2024-45492",
            "https://errata.almalinux.org/9/ALSA-2024-6754.html",
            "https://errata.rockylinux.org/RLSA-2024:6754",
            "https://github.com/libexpat/libexpat/blob/R_2_6_3/expat/Changes",
            "https://github.com/libexpat/libexpat/issues/887",
            "https://github.com/libexpat/libexpat/pull/890",
            "https://linux.oracle.com/cve/CVE-2024-45490.html",
            "https://linux.oracle.com/errata/ELSA-2024-6989.html",
            "https://nvd.nist.gov/vuln/detail/CVE-2024-45490",
            "https://security.netapp.com/advisory/ntap-20241018-0004/",
            "https://ubuntu.com/security/notices/USN-7000-1",
            "https://ubuntu.com/security/notices/USN-7000-2",
            "https://ubuntu.com/security/notices/USN-7001-1",
            "https://ubuntu.com/security/notices/USN-7001-2",
            "https://www.cve.org/CVERecord?id=CVE-2024-45490"
          ],
          "PublishedDate": "2024-08-30T03:15:03.757Z",
          "LastModifiedDate": "2025-03-14T19:15:47.253Z"
        },
        {
          "VulnerabilityID": "CVE-2024-8176",
          "PkgID": "libexpat@2.6.2-r0",
          "PkgName": "libexpat",
          "PkgIdentifier": {
            "PURL": "pkg:apk/alpine/libexpat@2.6.2-r0?arch=x86_64\u0026distro=3.19.1"
          },
          "InstalledVersion": "2.6.2-r0",
          "FixedVersion": "2.7.0-r0",
          "Status": "fixed",
          "Layer": {
            "DiffID": "sha256:80fef791f8cf38d8141a2963e4c83b8f5432c677137c657178d84c55f77e1476"
          },
          "PrimaryURL": "https://avd.aquasec.com/nvd/cve-2024-8176",
          "DataSource": {
            "ID": "alpine",
            "Name": "Alpine Secdb",
            "URL": "https://secdb.alpinelinux.org/"
          },
          "Title": "libexpat: expat: Improper Restriction of XML Entity Expansion Depth in libexpat",
          "Description": "A stack overflow vulnerability exists in the libexpat library due to the way it handles recursive entity expansion in XML documents. When parsing an XML document with deeply nested entity references, libexpat can be forced to recurse indefinitely, exhausting the stack space and causing a crash. This issue could lead to denial of service (DoS) or, in some cases, exploitable memory corruption, depending on the environment and library usage.",
          "Severity": "HIGH",
          "CweIDs": [
            "CWE-674"
          ],
          "VendorSeverity": {
            "alma": 2,
            "azure": 3,
            "cbl-mariner": 3,
            "oracle-oval": 2,
            "photon": 3,
            "redhat": 2,
            "rocky": 2,
            "ubuntu": 2
          },
          "CVSS": {
            "redhat": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
              "V3Score": 7.5
            }
          },
          "References": [
            "http://www.openwall.com/lists/oss-security/2025/03/15/1",
            "https://access.redhat.com/errata/RHSA-2025:13681",
            "https://access.redhat.com/errata/RHSA-2025:3531",
            "https://access.redhat.com/errata/RHSA-2025:3734",
            "https://access.redhat.com/errata/RHSA-2025:3913",
            "https://access.redhat.com/errata/RHSA-2025:4048",
            "https://access.redhat.com/errata/RHSA-2025:4446",
            "https://access.redhat.com/errata/RHSA-2025:4447",
            "https://access.redhat.com/errata/RHSA-2025:4448",
            "https://access.redhat.com/errata/RHSA-2025:4449",
            "https://access.redhat.com/errata/RHSA-2025:7444",
            "https://access.redhat.com/errata/RHSA-2025:7512",
            "https://access.redhat.com/errata/RHSA-2025:8385",
            "https://access.redhat.com/security/cve/CVE-2024-8176",
            "https://blog.hartwork.org/posts/expat-2-7-0-released/",
            "https://bugzilla.redhat.com/2310137",
            "https://bugzilla.redhat.com/show_bug.cgi?id=2310137",
            "https://bugzilla.suse.com/show_bug.cgi?id=1239618",
            "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2024-8176",
            "https://errata.almalinux.org/9/ALSA-2025-7444.html",
            "https://errata.rockylinux.org/RLSA-2025:4048",
            "https://github.com/libexpat/libexpat/blob/R_2_7_0/expat/Changes#L40-L52",
            "https://github.com/libexpat/libexpat/issues/893",
            "https://gitlab.alpinelinux.org/alpine/aports/-/commit/d068c3ff36fc6f4789988a09c69b434db757db53",
            "https://linux.oracle.com/cve/CVE-2024-8176.html",
            "https://linux.oracle.com/errata/ELSA-2025-7512.html",
            "https://nvd.nist.gov/vuln/detail/CVE-2024-8176",
            "https://security-tracker.debian.org/tracker/CVE-2024-8176",
            "https://security.netapp.com/advisory/ntap-20250328-0009/",
            "https://ubuntu.com/security/CVE-2024-8176",
            "https://ubuntu.com/security/notices/USN-7424-1",
            "https://www.cve.org/CVERecord?id=CVE-2024-8176",
            "https://www.kb.cert.org/vuls/id/760160"
          ],
          "PublishedDate": "2025-03-14T09:15:14.157Z",
          "LastModifiedDate": "2025-08-14T16:15:31.497Z"
        },
        {
          "VulnerabilityID": "CVE-2024-6119",
          "PkgID": "libssl3@3.1.4-r6",
          "PkgName": "libssl3",
          "PkgIdentifier": {
            "PURL": "pkg:apk/alpine/libssl3@3.1.4-r6?arch=x86_64\u0026distro=3.19.1"
          },
          "InstalledVersion": "3.1.4-r6",
          "FixedVersion": "3.1.7-r0",
          "Status": "fixed",
          "Layer": {
            "DiffID": "sha256:80fef791f8cf38d8141a2963e4c83b8f5432c677137c657178d84c55f77e1476"
          },
          "SeveritySource": "nvd",
          "PrimaryURL": "https://avd.aquasec.com/nvd/cve-2024-6119",
          "DataSource": {
            "ID": "alpine",
            "Name": "Alpine Secdb",
            "URL": "https://secdb.alpinelinux.org/"
          },
          "Title": "openssl: Possible denial of service in X.509 name checks",
          "Description": "Issue summary: Applications performing certificate name checks (e.g., TLS\nclients checking server certificates) may attempt to read an invalid memory\naddress resulting in abnormal termination of the application process.\n\nImpact summary: Abnormal termination of an application can a cause a denial of\nservice.\n\nApplications performing certificate name checks (e.g., TLS clients checking\nserver certificates) may attempt to read an invalid memory address when\ncomparing the expected name with an `otherName` subject alternative name of an\nX.509 certificate. This may result in an exception that terminates the\napplication program.\n\nNote that basic certificate chain validation (signatures, dates, ...) is not\naffected, the denial of service can occur only when the application also\nspecifies an expected DNS name, Email address or IP address.\n\nTLS servers rarely solicit client certificates, and even when they do, they\ngenerally don't perform a name check against a reference identifier (expected\nidentity), but rather extract the presented identity after checking the\ncertificate chain.  So TLS servers are generally not affected and the severity\nof the issue is Moderate.\n\nThe FIPS modules in 3.3, 3.2, 3.1 and 3.0 are not affected by this issue.",
          "Severity": "HIGH",
          "CweIDs": [
            "CWE-843"
          ],
          "VendorSeverity": {
            "alma": 2,
            "amazon": 2,
            "azure": 3,
            "cbl-mariner": 3,
            "nvd": 3,
            "oracle-oval": 2,
            "photon": 3,
            "redhat": 2,
            "rocky": 2,
            "ubuntu": 2
          },
          "CVSS": {
            "nvd": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
              "V3Score": 7.5
            },
            "redhat": {
              "V3Vector": "CVSS:3.1/AV:N/AC:H/PR:N/UI:N/S:U/C:N/I:N/A:H",
              "V3Score": 5.9
            }
          },
          "References": [
            "http://www.openwall.com/lists/oss-security/2024/09/03/4",
            "https://access.redhat.com/errata/RHSA-2024:8935",
            "https://access.redhat.com/security/cve/CVE-2024-6119",
            "https://bugzilla.redhat.com/2306158",
            "https://bugzilla.redhat.com/show_bug.cgi?id=2306158",
            "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2024-6119",
            "https://errata.almalinux.org/9/ALSA-2024-8935.html",
            "https://errata.rockylinux.org/RLSA-2024:6783",
            "https://github.com/openssl/openssl/commit/05f360d9e849a1b277db628f1f13083a7f8dd04f",
            "https://github.com/openssl/openssl/commit/06d1dc3fa96a2ba5a3e22735a033012aadc9f0d6",
            "https://github.com/openssl/openssl/commit/621f3729831b05ee828a3203eddb621d014ff2b2",
            "https://github.com/openssl/openssl/commit/7dfcee2cd2a63b2c64b9b4b0850be64cb695b0a0",
            "https://github.com/openssl/openssl/security/advisories/GHSA-5qrj-vq78-58fj",
            "https://linux.oracle.com/cve/CVE-2024-6119.html",
            "https://linux.oracle.com/errata/ELSA-2024-8935.html",
            "https://lists.freebsd.org/archives/freebsd-security/2024-September/000303.html",
            "https://nvd.nist.gov/vuln/detail/CVE-2024-6119",
            "https://openssl-library.org/news/secadv/20240903.txt",
            "https://security.netapp.com/advisory/ntap-20240912-0001/",
            "https://ubuntu.com/security/notices/USN-6986-1",
            "https://www.cve.org/CVERecord?id=CVE-2024-6119"
          ],
          "PublishedDate": "2024-09-03T16:15:07.177Z",
          "LastModifiedDate": "2025-06-03T10:51:54.117Z"
        },
        {
          "VulnerabilityID": "CVE-2025-29087",
          "PkgID": "sqlite-libs@3.44.2-r0",
          "PkgName": "sqlite-libs",
          "PkgIdentifier": {
            "PURL": "pkg:apk/alpine/sqlite-libs@3.44.2-r0?arch=x86_64\u0026distro=3.19.1"
          },
          "InstalledVersion": "3.44.2-r0",
          "FixedVersion": "3.44.2-r1",
          "Status": "fixed",
          "Layer": {
            "DiffID": "sha256:80fef791f8cf38d8141a2963e4c83b8f5432c677137c657178d84c55f77e1476"
          },
          "SeveritySource": "nvd",
          "PrimaryURL": "https://avd.aquasec.com/nvd/cve-2025-29087",
          "DataSource": {
            "ID": "alpine",
            "Name": "Alpine Secdb",
            "URL": "https://secdb.alpinelinux.org/"
          },
          "Title": "sqlite: Integer Overflow in SQLite concat_ws Function",
          "Description": "In SQLite 3.44.0 through 3.49.0 before 3.49.1, the concat_ws() SQL function can cause memory to be written beyond the end of a malloc-allocated buffer. If the separator argument is attacker-controlled and has a large string (e.g., 2MB or more), an integer overflow occurs in calculating the size of the result buffer, and thus malloc may not allocate enough memory.",
          "Severity": "HIGH",
          "CweIDs": [
            "CWE-190"
          ],
          "VendorSeverity": {
            "bitnami": 3,
            "nvd": 3,
            "redhat": 2,
            "ubuntu": 2
          },
          "CVSS": {
            "bitnami": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
              "V3Score": 7.5
            },
            "nvd": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
              "V3Score": 7.5
            },
            "redhat": {
              "V3Vector": "CVSS:3.1/AV:L/AC:L/PR:L/UI:N/S:U/C:N/I:N/A:H",
              "V3Score": 5.5
            }
          },
          "References": [
            "https://access.redhat.com/security/cve/CVE-2025-29087",
            "https://gist.github.com/ylwango613/a44a29f1ef074fa783e29f04a0afd62a",
            "https://nvd.nist.gov/vuln/detail/CVE-2025-29087",
            "https://sqlite.org/releaselog/3_49_1.html",
            "https://ubuntu.com/security/notices/USN-7528-1",
            "https://www.cve.org/CVERecord?id=CVE-2025-29087",
            "https://www.sqlite.org/cves.html"
          ],
          "PublishedDate": "2025-04-07T20:15:20.253Z",
          "LastModifiedDate": "2025-04-30T12:43:22.31Z"
        },
        {
          "VulnerabilityID": "CVE-2025-31115",
          "PkgID": "xz-libs@5.4.5-r0",
          "PkgName": "xz-libs",
          "PkgIdentifier": {
            "PURL": "pkg:apk/alpine/xz-libs@5.4.5-r0?arch=x86_64\u0026distro=3.19.1"
          },
          "InstalledVersion": "5.4.5-r0",
          "FixedVersion": "5.4.5-r1",
          "Status": "fixed",
          "Layer": {
            "DiffID": "sha256:80fef791f8cf38d8141a2963e4c83b8f5432c677137c657178d84c55f77e1476"
          },
          "PrimaryURL": "https://avd.aquasec.com/nvd/cve-2025-31115",
          "DataSource": {
            "ID": "alpine",
            "Name": "Alpine Secdb",
            "URL": "https://secdb.alpinelinux.org/"
          },
          "Title": "xz: XZ has a heap-use-after-free bug in threaded .xz decoder",
          "Description": "XZ Utils provide a general-purpose data-compression library plus command-line tools. In XZ Utils 5.3.3alpha to 5.8.0, the multithreaded .xz decoder in liblzma has a bug where invalid input can at least result in a crash. The effects include heap use after free and writing to an address based on the null pointer plus an offset. Applications and libraries that use the lzma_stream_decoder_mt function are affected. The bug has been fixed in XZ Utils 5.8.1, and the fix has been committed to the v5.4, v5.6, v5.8, and master branches in the xz Git repository. No new release packages will be made from the old stable branches, but a standalone patch is available that applies to all affected releases.",
          "Severity": "HIGH",
          "CweIDs": [
            "CWE-366",
            "CWE-416",
            "CWE-476",
            "CWE-826"
          ],
          "VendorSeverity": {
            "alma": 3,
            "azure": 3,
            "oracle-oval": 3,
            "photon": 3,
            "redhat": 3,
            "ubuntu": 2
          },
          "CVSS": {
            "redhat": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
              "V3Score": 7.5
            }
          },
          "References": [
            "http://www.openwall.com/lists/oss-security/2025/04/03/1",
            "http://www.openwall.com/lists/oss-security/2025/04/03/2",
            "http://www.openwall.com/lists/oss-security/2025/04/03/3",
            "https://access.redhat.com/errata/RHSA-2025:7524",
            "https://access.redhat.com/security/cve/CVE-2025-31115",
            "https://bugzilla.redhat.com/2357249",
            "https://errata.almalinux.org/10/ALSA-2025-7524.html",
            "https://github.com/tukaani-project/xz/commit/d5a2ffe41bb77b918a8c96084885d4dbe4bf6480",
            "https://github.com/tukaani-project/xz/security/advisories/GHSA-6cc8-p5mm-29w2",
            "https://linux.oracle.com/cve/CVE-2025-31115.html",
            "https://linux.oracle.com/errata/ELSA-2025-7524.html",
            "https://nvd.nist.gov/vuln/detail/CVE-2025-31115",
            "https://tukaani.org/xz/xz-cve-2025-31115.patch",
            "https://ubuntu.com/security/notices/USN-7414-1",
            "https://www.cve.org/CVERecord?id=CVE-2025-31115"
          ],
          "PublishedDate": "2025-04-03T17:15:30.54Z",
          "LastModifiedDate": "2025-04-07T14:18:34.453Z"
        }
      ]
    },
    {
      "Target": "Python",
      "Class": "lang-pkgs",
      "Type": "python-pkg",
      "Vulnerabilities": [
        {
          "VulnerabilityID": "CVE-2024-6345",
          "PkgName": "setuptools",
          "PkgPath": "usr/local/lib/python3.12/site-packages/setuptools-69.2.0.dist-info/METADATA",
          "PkgIdentifier": {
            "PURL": "pkg:pypi/setuptools@69.2.0"
          },
          "InstalledVersion": "69.2.0",
          "FixedVersion": "70.0.0",
          "Status": "fixed",
          "Layer": {
            "DiffID": "sha256:d46b5001af7f7573102bc997debe285c1d57558094abbd2d66b8399e1dea4a67"
          },
          "SeveritySource": "ghsa",
          "PrimaryURL": "https://avd.aquasec.com/nvd/cve-2024-6345",
          "DataSource": {
            "ID": "ghsa",
            "Name": "GitHub Security Advisory pip",
            "URL": "https://github.com/advisories?query=type%3Areviewed+ecosystem%3Apip"
          },
          "Title": "pypa/setuptools: Remote code execution via download functions in the package_index module in pypa/setuptools",
          "Description": "A vulnerability in the package_index module of pypa/setuptools versions up to 69.1.1 allows for remote code execution via its download functions. These functions, which are used to download packages from URLs provided by users or retrieved from package index servers, are susceptible to code injection. If these functions are exposed to user-controlled inputs, such as package URLs, they can execute arbitrary commands on the system. The issue is fixed in version 70.0.",
          "Severity": "HIGH",
          "CweIDs": [
            "CWE-94"
          ],
          "VendorSeverity": {
            "alma": 3,
            "amazon": 3,
            "azure": 3,
            "bitnami": 3,
            "cbl-mariner": 3,
            "ghsa": 3,
            "oracle-oval": 3,
            "photon": 3,
            "redhat": 3,
            "rocky": 3,
            "ubuntu": 2
          },
          "CVSS": {
            "bitnami": {
              "V3Vector": "CVSS:3.0/AV:N/AC:L/PR:N/UI:R/S:U/C:H/I:H/A:H",
              "V3Score": 8.8
            },
            "ghsa": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:R/S:U/C:H/I:H/A:H",
              "V3Score": 8.8
            },
            "redhat": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:R/S:U/C:H/I:H/A:H",
              "V3Score": 8.8
            }
          },
          "References": [
            "https://access.redhat.com/errata/RHSA-2024:6726",
            "https://access.redhat.com/security/cve/CVE-2024-6345",
            "https://bugzilla.redhat.com/2297771",
            "https://bugzilla.redhat.com/show_bug.cgi?id=2297771",
            "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2024-6345",
            "https://errata.almalinux.org/9/ALSA-2024-6726.html",
            "https://errata.rockylinux.org/RLSA-2024:6726",
            "https://github.com/pypa/setuptools",
            "https://github.com/pypa/setuptools/commit/88807c7062788254f654ea8c03427adc859321f0",
            "https://github.com/pypa/setuptools/pull/4332",
            "https://huntr.com/bounties/d6362117-ad57-4e83-951f-b8141c6e7ca5",
            "https://linux.oracle.com/cve/CVE-2024-6345.html",
            "https://linux.oracle.com/errata/ELSA-2024-6726.html",
            "https://nvd.nist.gov/vuln/detail/CVE-2024-6345",
            "https://ubuntu.com/security/notices/USN-7002-1",
            "https://www.cve.org/CVERecord?id=CVE-2024-6345"
          ],
          "PublishedDate": "2024-07-15T01:15:01.73Z",
          "LastModifiedDate": "2024-11-21T09:49:28.17Z"
        },
        {
          "VulnerabilityID": "CVE-2025-47273",
          "PkgName": "setuptools",
          "PkgPath": "usr/local/lib/python3.12/site-packages/setuptools-69.2.0.dist-info/METADATA",
          "PkgIdentifier": {
            "PURL": "pkg:pypi/setuptools@69.2.0"
          },
          "InstalledVersion": "69.2.0",
          "FixedVersion": "78.1.1",
          "Status": "fixed",
          "Layer": {
            "DiffID": "sha256:d46b5001af7f7573102bc997debe285c1d57558094abbd2d66b8399e1dea4a67"
          },
          "SeveritySource": "ghsa",
          "PrimaryURL": "https://avd.aquasec.com/nvd/cve-2025-47273",
          "DataSource": {
            "ID": "ghsa",
            "Name": "GitHub Security Advisory pip",
            "URL": "https://github.com/advisories?query=type%3Areviewed+ecosystem%3Apip"
          },
          "Title": "setuptools: Path Traversal Vulnerability in setuptools PackageIndex",
          "Description": "setuptools is a package that allows users to download, build, install, upgrade, and uninstall Python packages. A path traversal vulnerability in `PackageIndex` is present in setuptools prior to version 78.1.1. An attacker would be allowed to write files to arbitrary locations on the filesystem with the permissions of the process running the Python code, which could escalate to remote code execution depending on the context. Version 78.1.1 fixes the issue.",
          "Severity": "HIGH",
          "CweIDs": [
            "CWE-22"
          ],
          "VendorSeverity": {
            "alma": 2,
            "amazon": 3,
            "azure": 3,
            "bitnami": 3,
            "ghsa": 3,
            "nvd": 3,
            "oracle-oval": 2,
            "photon": 3,
            "redhat": 2,
            "rocky": 2,
            "ubuntu": 2
          },
          "CVSS": {
            "nvd": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:H/I:H/A:H",
              "V3Score": 8.8
            },
            "redhat": {
              "V3Vector": "CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:N/I:H/A:L",
              "V3Score": 7.1
            }
          },
          "References": [
            "https://access.redhat.com/errata/RHSA-2025:13578",
            "https://access.redhat.com/security/cve/CVE-2025-47273",
            "https://bugzilla.redhat.com/2366982",
            "https://bugzilla.redhat.com/show_bug.cgi?id=2366982",
            "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2025-47273",
            "https://errata.almalinux.org/9/ALSA-2025-13578.html",
            "https://errata.rockylinux.org/RLSA-2025:11036",
            "https://github.com/pypa/advisory-database/tree/main/vulns/setuptools/PYSEC-2025-49.yaml",
            "https://github.com/pypa/setuptools",
            "https://github.com/pypa/setuptools/blob/6ead555c5fb29bc57fe6105b1bffc163f56fd558/setuptools/package_index.py#L810C1-L825C88",
            "https://github.com/pypa/setuptools/commit/250a6d17978f9f6ac3ac887091f2d32886fbbb0b",
            "https://github.com/pypa/setuptools/issues/4946",
            "https://github.com/pypa/setuptools/security/advisories/GHSA-5rjg-fvgr-3xxf",
            "https://linux.oracle.com/cve/CVE-2025-47273.html",
            "https://linux.oracle.com/errata/ELSA-2025-9940.html",
            "https://lists.debian.org/debian-lts-announce/2025/05/msg00035.html",
            "https://nvd.nist.gov/vuln/detail/CVE-2025-47273",
            "https://ubuntu.com/security/notices/USN-7544-1",
            "https://www.cve.org/CVERecord?id=CVE-2025-47273"
          ],
          "PublishedDate": "2025-05-17T16:15:19.11Z",
          "LastModifiedDate": "2025-06-12T16:29:01.66Z"
        }
      ]
    }
  ]
}
